src/main/scala/org/broadinstitute/dsde/vault/services/generic/GenericIngestService.scala

```diff
@@ -0,0 +1,57 @@
+package org.broadinstitute.dsde.vault.services.generic
+
+import com.wordnik.swagger.annotations._
+import org.broadinstitute.dsde.vault.common.directives.VersioningDirectives._
+import org.broadinstitute.dsde.vault.model.{GenericSysAttrs, GenericEntity, GenericIngest}
+import org.broadinstitute.dsde.vault.model.GenericJsonProtocol._
+import spray.json._
+import spray.http.MediaTypes._
+import spray.httpx.SprayJsonSupport._
+import spray.routing.HttpService
+
+@Api(value="/entities", description="generic entity service", produces="application/json")
+trait GenericIngestService extends HttpService {
+  private final val ApiPrefix = "entities"
+  private final val ApiVersions = "v1"
+  private final val DefaultVersion = 1
+
+  val giRoute = ingestRoute
+
+  @ApiOperation(
+    value = "store some entities and relationships",
+    nickname = "genericIngest",
+    httpMethod = "POST",
+    response = classOf[GenericEntity],
+    responseContainer = "List",
+    notes = "response is a list of the new entities")
```

Comment:
This is necessary because of the PUT URLs: there's no way to see them except as the response to this ingest.

Alternately, it could respond with `[ { "guid", "url" }, { "guid", "url" } ]` but my preference is to keep the full entities for context.